casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -146,8 +162,13 @@ object ProtoUtil {
   def hashString(b: BlockMessage): String = Base16.encode(b.blockHash.toByteArray)
 
   def basicDeploy(id: Int): Deploy = {
+    val term = InterpreterUtil.mkTerm(s"@${id}!($id)").right.get
```

Comment:
Is this just dummy code for the moment?  Otherwise I don't understand what this is attempting.

Reply:
Yeah, just a dummy. This is only used in the deploy demo and in tests as an easy way of generating Rholang terms. Real deployments are covered in #722